Rubberduck.Core/UI/Inspections/InspectionResultsViewModel.cs

```diff
@@ -202,136 +209,151 @@ public IEnumerable<DisplayQuickFix> QuickFixes
             }
         }
 
-        private bool _groupByInspectionType = true;
-        public bool GroupByInspectionType
+        private static readonly Dictionary<InspectionResultGrouping, PropertyGroupDescription> GroupDescriptions = new Dictionary<InspectionResultGrouping, PropertyGroupDescription>
         {
-            get => _groupByInspectionType;
-            set
-            {
-                if (_groupByInspectionType == value) { return; }
+            { InspectionResultGrouping.Type, new PropertyGroupDescription("Inspection", new InspectionTypeConverter()) },
+            { InspectionResultGrouping.Name, new PropertyGroupDescription("Inspection.Name") },
+            { InspectionResultGrouping.Location, new PropertyGroupDescription("QualifiedSelection.QualifiedName") },
+            { InspectionResultGrouping.Severity, new PropertyGroupDescription("Inspection.Severity") }
```

Comment:
Do they have to be literals? Const? Resource?

Reply:
`PropertyGroupDescription` takes a string for the first argument (it's the equivalent to the xaml `Path` on a binding). It seems silly to make a second lookup set with the same keys as this one.